### ISSUE TITLE
php 7 and tableExists()

### ISSUE THREAD

**Issue by Stewe21** [joshcam/PHP-MySQLi-Database-Class#443]
tableExists("example") not working with PHP 7?

**Comment by avbdr** [comment]
what error you are actually getting?

**Comment by somethingweird** [comment]
I also having that issue - 

PHP 7.0 from remi repo using php-fpm

Fatal error: Uncaught Exception: Problem preparing query (SELECT * FROM information_schema.tables WHERE table_schema = NULL AND table_name NULL LIMIT 1) 

You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'NULL LIMIT 1' at line 1

Side Note:
Notice: Undefined index: in .../vendor/joshcam/mysqli-database-class/MysqliDb.php on line 732

Version I'm using..
  - Updating joshcam/mysqli-database-class dev-master (00824c1 => 95b27c5)
    Checking out 95b27c5af0772bb6df3f93d7bcd11787daedd79a

**Comment by webmonkey100** [comment]
I also have the issue (uwamp/php 7.0.3/class 2.6)
----------
Fatal error: Uncaught Exception: Problem preparing query (SELECT * FROM information_schema.tables WHERE table_schema = ? AND table_name NULL LIMIT 1) You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'NULL LIMIT 1' at line 1 in .....mysqlidb.php on line 1482

**Comment by chenmin201** [comment]
I have found the cause of this problem!

Problem fixed after I did the following changes in MysqliDb.php:

```
public function where($whereProp, $whereValue = 'DBNULL', $operator = '=', $cond = 'AND') {
	// forkaround for an old operation api

**	// reset the internal pointer of the $whereValue array. This fix problems in PHP 7
	if (is_array($whereValue)) {reset($whereValue);}**

	if (is_array($whereValue) && ($key = key($whereValue)) != "0") {
		$operator = $key;
		$whereValue = $whereValue[$key];
	}
	if (count($this->_where) == 0) {
		$cond = '';
	}

	$this->_where[] = Array($cond, $whereProp, $operator, $whereValue);
	return $this;
}
```

It seems like, somehow, in PHP 7, when $whereValue array is passed into function where(), the array's internal pointer has already been advanced to the end of the array. key($whereValue) would actually return NULL.

**Comment by avbdr** [comment]
thats fixed in a latest release 2.6. Thank you for your report.
